nipype/interfaces/freesurfer/base.py

```diff
@@ -151,11 +195,28 @@ def _gen_fname(self, basename, fname=None, cwd=None, suffix='_fs',
                                 use_ext=use_ext, newpath=cwd)
         return fname
 
-    @property
-    def version(self):
-        ver = Info.version()
-        if ver:
-            if 'dev' in ver:
-                return ver.rstrip().split('-')[-1] + '.dev'
-            else:
-                return ver.rstrip().split('-v')[-1]
+
+class FSScriptCommand(FSCommand):
+    """ Support for Freesurfer script commands with log inputs.terminal_output """
+    _terminal_output = 'file'
+    _always_run = False
+
+    def __init__(self, **inputs):
+        super(FSScriptCommand, self).__init__(**inputs)
+        self.set_default_terminal_output(self._terminal_output)
+
+    def _list_outputs(self):
+        outputs = self.output_spec().get()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [self.output_spec is not callable](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/825382731/issues/source?bid=2767207&fileBranchId=2768049#l209)